packages/api/config/api_default.yml

```diff
@@ -0,0 +1,383 @@
+ApiCollectionsDefault:
```

Comment:
For my own edification what is the difference between `api_default` and `api_v1`?